<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Log in</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="error">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
   <property name="text">
    <string>Invalid name or password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
